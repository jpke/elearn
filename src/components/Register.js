import React from 'react';

const Register = (props) => {
  return (
    <div className="quizInProgress">
      <h3 className="authTitle">Register</h3>
      <form onSubmit={props.register} id="authRegister">
        <input type="text" id="userName" className="auth" placeholder="name" />
        <input type="email" id="email" className="auth" placeholder="email" />
        <input type="password" id="password" className="auth" placeholder="password" />
        <button type="submit">Register</button>
      </form>
    </div>
  )
};

export default Register;
