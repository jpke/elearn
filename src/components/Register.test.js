/* eslint-disable no-undef */
/* eslint-disable import/no-named-as-default */

import React from 'react';
import renderer from 'react-test-renderer';

import Register from './Register';

it('renders correctly', () => {
  const tree = renderer.create(
    <Register register={jest.fn()} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
