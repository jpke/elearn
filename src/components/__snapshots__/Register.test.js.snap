exports[`test renders correctly 1`] = `
<div
  className="registerContainer">
  <h3
    className="title">
    Register
  </h3>
  <form
    id="authForm"
    onSubmit={[Function]}>
    <input
      className="auth"
      id="userName"
      placeholder="name"
      type="text" />
    <input
      className="auth"
      id="email"
      placeholder="email"
      type="email" />
    <input
      className="auth"
      id="password"
      placeholder="password"
      type="password" />
    <button
      className="authButton"
      type="submit">
      Register
    </button>
  </form>
</div>
`;
